content/Solutions/Integrations/SMBClustering.md

```diff
@@ -101,30 +120,64 @@ Click **CREATE SHARE**.
 1. Enter a unique **Name** for the share.
 2. Choose an **ACL** to apply to the share.
    {{< expand "Current Options (click to expand)" "v" >}}
-   * **POSIX_OPEN** - Template that grants read, write, and execute permissions to all users.
-   * **POSIX_RESTRICTED** - Template that grants read, write, and execute to owner and group, but not other. The template may optionally include the special-purpose 'builtin_users' and 'builtin_administrators' groups as well as Domain Users and Domain Admins groups in Active Directory environments.
+   * **POSIX_OPEN** - Grants read, write, and execute permissions for all users.
+   * **POSIX_RESTRICTED** - Grants read, write, and execute to owner and group, but not others. The template may optionally include the special-purpose 'builtin_users' and 'builtin_administrators' groups and Domain Users and Domain Admins groups in Active Directory environments.
    {{< /expand >}}
-3. Setting **Readonly** prevents users from making any changes to the cluster volume contents.
+3. Setting **Readonly** prevents users from changing the cluster volume contents.
 4. Click **CONFIRM** to create the SMB share and make it immediately active.
 
 {{< hint info >}}
-The SMB share is added to the SCALE **Shares** > **SMB** section for each system in the cluster.
+The SMB share adds to the SCALE **Shares** > **SMB** section for each system in the cluster.
 Attempting to manage the share from the SCALE UI is not recommended.
 {{< /hint >}}
 
 ### Connecting to the Shared Volume
 
-There are many different ways to access an SMB share, but this article demonstrates using the Windows 10 File Explorer.
+There are several ways to access an SMB share, but this article demonstrates using Windows 10 File Explorer.
 
-1. From a Windows 10 system that is connected to the same network as the clustering environment, open File Explorer.
+1. From a Windows 10 system connected to the same network as the clustering environment, open File Explorer.
```

Comment:
On File Explorer, we either need to put it in bold because it is the UI name for it, or use lower case and not bother with caps. I don't have strong feelings about it one way or the other.
```suggestion
1. From a Windows 10 system connected to the same network as the clustering environment, open **File Explorer**.
```